src/platform/packages/shared/kbn-scout/README.md

```diff
@@ -154,10 +171,112 @@ test.beforeEach(async ({ pageObjects }) => {
+
+#### Setting up Test Directory
+
+To get started with Scout testing for your plugin, you need to create the appropriate directory structure in your plugin's root directory:
+
+```
+your-plugin/
+├── test/
+│   └── scout/
+│       ├── ui/
+│       │   ├── playwright.config.ts
+│       │   ├── parallel.playwright.config.ts
+│       │   └── parallel_tests/               # Your UI test specs, that are run in parallel
+│       │   └── tests/                        # Your UI test specs, that are run sequentially
+│       └── api/
+│           ├── playwright.config.ts
+│           └── tests/                        # Your API test specs, that are run sequentially
+```
+
+#### UI Tests
+
+UI tests are designed for browser-based integration testing and provide access to browser fixtures like `page`, `pageObjects`, and `browserAuth`.
+
+**Test Imports for UI Testing:**
+
+```ts
+// For sequential UI tests
+import { test, expect } from '@kbn/scout';
+
+// For parallel UI tests that can be space-isolated
+import { spaceTest as test, expect } from '@kbn/scout';
+```
+
+**When to use each:**
+
+- **`spaceTest`**: Use for parallel tests that can be isolated by Kibana spaces, allowing faster execution
+- **`test`**: Use for sequential tests that cannot run in parallel
+
+**Example UI Test:**
+
+```ts
+import { spaceTest as test, expect } from '@kbn/scout';
+
+test('should display dashboard', async ({ pageObjects, page }) => {
+  await pageObjects.dashboard.goto();
+  await expect(page.testSubj.locator('dashboardLandingPage')).toBeVisible();
+});
+```
+
+#### API Tests
+
+API tests are designed for server-side testing and provide fixtures focused on API interactions without browser-related fixtures.
+
+**Test Import for API Testing:**
+
+```ts
+// For API integration tests (server-side only, no browser fixtures)
+import { apiTest as test, expect } from '@kbn/scout';
+```
+
+**Example API Test:**
+
+```ts
+import { apiTest, expect } from '@kbn/scout';
+
+apiTest('POST api/painless_lab/execute is disabled', async ({ kbnClient, log }) => {
+  const response = await apiClient.post('api/painless_lab/execute', {
+    headers: {
+      ...COMMON_HEADERS,
+      ...adminApiCredentials.apiKeyHeader,
+    },
+    responseType: 'json',
+    body: TEST_INPUT.script,
+  });
+  expect(response.statusCode).toBe(404);
+});
+```
+
+**Key Differences:**
+
+- **UI tests** include browser fixtures (`page`, `pageObjects`, `browserAuth`) for UI interactions
+- **API tests** exclude browser fixtures and focus on server-side operations (`kbnClient`, `esClient`, `log`, etc.)
+
+#### Testing Guidelines for Plugin Development
+
+When writing tests for your plugin, consider the following guidelines to ensure comprehensive coverage:
+
+**Focus on Plugin Functionality:**
+
+- Tests should primarily cover the specific functionality exposed by your plugin
+- Focus on plugin-specific user workflows, configurations, and integrations
+
+**API Testing Coverage:**
```

Comment:
Some small typos in the page:

* `keep in naming` should likely be `keep mind in naming`
* `sharable` should be `shareable`